src/controllers/queries/index.ts

```diff
@@ -403,7 +518,7 @@ export class QueriesController extends HTMLController {
     connectionId: UUID,
     parentId: UUID | null,
     params: Omit<SubmitQueryRequest['params'], 'id' | 'createdTime' | 'expiresTime'>,
-    expiresAt: Date
+    expiresTime: Date
```

Comment:
porque?

Reply:
across the repo it's either `expiresAt` or `expiresTime`, there was more uses of `expiresTime` so I made it like that everywhere (although the `db` column is actually called `expires_at`, that's harder to change)